File: Python Excel Examples/PropertiesApiDemo/deleteDocumentProperty.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.properties_api import PropertiesApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.properties_api.PropertiesApi(configuration)
name = "DeleteDocumentProperty_Title.xlsx"
password = ""
folder = "Properties"
storage = ""
propertyName = "Title"
api.delete_document_property(name, propertyName, password=password, folder=folder, storage=storage)

File: PHP Excel Examples/PropertiesApiDemo/deleteDocumentProperty.php
<?php

use Spire\Cloud\Excel\Sdk\Api\PropertiesApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new PropertiesApi($configuration);

$name = "DeleteDocumentProperty_Title.xlsx";
$folder = "input";
$password = null;
$storage = null;
$propertyName = "Title";
$apiInstance->deleteDocumentProperty($name, $propertyName, $password, $folder, $storage);
